packages/react-native-aztec/android/build.gradle

```diff
@@ -120,37 +117,28 @@ dependencies {
 }
 
 if (willPublishReactNativeAztecBinary) {
-    bintray {
-        user = project.hasProperty('bintrayUser') ? project.property('bintrayUser') : System.getenv('BINTRAY_USER')
-        key = project.hasProperty('bintrayKey') ? project.property('bintrayKey') : System.getenv('BINTRAY_KEY')
-        publications = ['ReactNativeAztecPublication']
-        publish = true
-        pkg {
-            repo = 'maven'
-            name = 'react-native-aztec'
-            licenses = ['GPL-2.0']
-            userOrg = 'wordpress-mobile'
-            vcsUrl = 'https://github.com/wordpress-mobile/gutenberg-mobile.git'
-            version {
-                name = publishReactNativeAztecVersion
-                released  = new Date()
-            }
-        }
-    }
-
     project.afterEvaluate {
         publishing {
             publications {
                 ReactNativeAztecPublication(MavenPublication) {
                     artifact bundleReleaseAar
 
-                    groupId 'com.github.wordpress-mobile.gutenberg-mobile'
+                    groupId 'org.wordpress-mobile.gutenberg-mobile'
```

Comment:
Nit: can we reference the `group` variable declared above for this, or do they have to be separate?

Same comment for the other `build.gradle` file

Reply:
Good point - but let me address that as a follow up PR just so we can merge this one as soon as possible.